sql/src/lib.rs

```diff
@@ -103,7 +104,7 @@ pub struct SQLiteQueryBuilder {
     // Instead we track byte and String arguments separately, mapping them to their argument name,
     // in order to dedupe. We'll add these to the regular argument vector later.
     byte_args: HashMap<Vec<u8>, String>,             // From value to argument name.
-    string_args: HashMap<Rc<String>, String>,        // From value to argument name.
+    string_args: HashMap<Arc<String>, String>,        // From value to argument name.
```

Comment:
I see no reason why this would need to change. Did you just replace every `Rc` in the project, or did you make the changes necessary to pass some particular test?

Reply:
Some changes to Arc simplified the refactoring as most code with clone or cloned could remain the same instead of implementing some convert from Arc->Rc which would require to clone the inner value, so just cloning Arc potentially is cheaper then cloning the inner value into a new Rc.